src/cmd/cmd.cpp

```diff
@@ -93,10 +93,9 @@ void selectiveSyncFixup(OCC::SyncJournalDb *journal, const QStringList &newList)
 
     bool ok;
 
-    auto oldBlackListSet = journal->getSelectiveSyncList(SyncJournalDb::SelectiveSyncBlackList, &ok).toSet();
+    const auto oldBlackListSet = journal->getSelectiveSyncList(SyncJournalDb::SelectiveSyncBlackList, &ok);
```

Comment:
Now that both `oldBlackListSet` and `newList` have the same type, naming should be adjusted.

```suggestion
    const auto oldBlackList = journal->getSelectiveSyncList(SyncJournalDb::SelectiveSyncBlackList, &ok);
```